### ISSUE TITLE
Landmark detection accuracy?

### ISSUE THREAD

**Issue by ModarD** [yinguobing/head-pose-estimation#21]
Hello, thank you for the great code! I wonder if the landmark detection accuracy in this project is identical to the one in this project https://github.com/yinguobing/cnn-facial-landmark
is it the same CNN and training dataset?

Do you have any suggestions for getting more accurate frontal face landmarks detection for eyes closed and mouth is widely open or one eyebrow is higher than the other one?

Thanks!

**Comment by yinguobing** [comment]
For the first question the answer is yes.

For the second question, this tutorial is for beginners so the model and the loss function are simplified. If accuracy is a key requirement you may want to refer to some state of the art technical/academic papers. This project is not the best option.
